CMakeLists.txt

```diff
@@ -239,6 +240,12 @@ if(PCAPPP_INSTALL)
 
   if(APPLE)
     set(CPACK_SYSTEM_NAME "Darwin-${CMAKE_SYSTEM_PROCESSOR}")
+  elseif(LINUX)
+    # Easier for Packing but requires CMake 3.22
```

Comment:
@seladb what do you think about requirering a recent cmake version for packaging? 
I can make this limited to Package only.
So user can still build and install with old version of cmake

Reply:
I'm not sure I understand the difference...

Yes, I'd like people to be able to build and install PcapPlusPlus with the oldest version of CMake we can allow

Reply:
This mean that I case we want to package we have to add -DPCAPP_ENABLE_PACKAGE=ON

Reply:
see: https://github.com/seladb/PcapPlusPlus/pull/1074/commits/5251c0c3c3ce91dea340db4ee2f8740973a9bf81

Reply:
So can we allow an older CMake version for packaging and a newer version for building? 🤔

Reply:
older for building and installing, newer for building, installing and packaging

Reply:
ok sounds good!